index.pug

```diff
@@ -0,0 +1,96 @@
+html
```

Comment:
This needs to be in a `views` folder for Express to find it by name.

![image](https://user-images.githubusercontent.com/5775716/81195597-0f1d6700-8f8c-11ea-8ee8-280fa75b609f.png)